<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editdataa</class>
 <widget class="QDialog" name="editdataa">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>561</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Data</string>
  </property>
  <property name="windowIcon">
   <iconset resource="misc.qrc">
    <normaloff>:/unsoed.png</normaloff>:/unsoed.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QDialog#editdataa {
	background-color: #ffc97c;
}</string>
  </property>
  <widget class="QLabel" name="label1">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>561</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel#label1 {
	font: 16pt &quot;Comic Sans MS&quot;;
	font-weight: bold;
	color: #fcf9f4;
}</string>
   </property>
   <property name="text">
    <string>EDIT DATA ANGGOTA</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>421</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label2">
        <property name="styleSheet">
         <string notr="true">QLabel#label2 {
	font: 10pt &quot;Comic Sans MS&quot;;
	font-weight: 500;
	color: #fcf9f4;
}</string>
        </property>
        <property name="text">
         <string>NO ID</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>68</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="noIDEdit">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QLineEdit#noIDEdit {
	border-radius: 5px;
	height: 30px;
	padding-left: 10px;
	font: 10pt &quot;Comic Sans MS&quot;;
}</string>
        </property>
        <property name="maxLength">
         <number>20</number>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="placeholderText">
         <string>No ID</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label3">
        <property name="styleSheet">
         <string notr="true">QLabel#label3 {
	font: 10pt &quot;Comic Sans MS&quot;;
	font-weight: 500;
	color: #fcf9f4;
}</string>
        </property>
        <property name="text">
         <string>NAMA</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>70</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="namaEdit">
        <property name="styleSheet">
         <string notr="true">QLineEdit#namaEdit{
	border-radius: 5px;
	height: 30px;
	padding-left: 10px;
	font: 10pt &quot;Comic Sans MS&quot;;
}</string>
        </property>
        <property name="placeholderText">
         <string>Nama</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label4">
        <property name="styleSheet">
         <string notr="true">QLabel#label4 {
	font: 10pt &quot;Comic Sans MS&quot;;
	font-weight: 500;
	color: #fcf9f4;
}</string>
        </property>
        <property name="text">
         <string>ALAMAT</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>53</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLineEdit" name="alamatEdit">
        <property name="styleSheet">
         <string notr="true">QLineEdit#alamatEdit {
	border-radius: 5px;
	height: 30px;
	padding-left: 10px;
	font: 10pt &quot;Comic Sans MS&quot;;
}</string>
        </property>
        <property name="placeholderText">
         <string>Alamat</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label5">
        <property name="styleSheet">
         <string notr="true">QLabel#label5 {
	font: 10pt &quot;Comic Sans MS&quot;;
	font-weight: 500;
	color: #fcf9f4;
}</string>
        </property>
        <property name="text">
         <string>FOTO</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>72</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="getFoto">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#getFoto {
	color: #a9a9a9;
	height: 30px;
	border-radius: 5px;
	background-color: #e1e1e1;
	font: 12pt &quot;Comic Sans MS&quot;;
	font-weight: bold;
}
QPushButton#getFoto:hover {
	background-color: #d1d1d1;
}</string>
        </property>
        <property name="text">
         <string>Pilih File</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label6">
        <property name="styleSheet">
         <string notr="true">QLabel#label6 {
	font: 10pt &quot;Comic Sans MS&quot;;
	font-weight: 500;
	color: #fcf9f4;
}</string>
        </property>
        <property name="text">
         <string>TIPE ANGGOTA</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>1</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QComboBox" name="tipeAnggotaPilih">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QComboBox#tipeAnggotaPilih {
	color: #a9a9a9;
	border-radius: 5px;
	background-color: #e1e1e1;
	height: 30px;
	padding-left: 10px;
	font: 12pt &quot;Comic Sans MS&quot;;
	height: 30px;
}
QComboBox#tipeAnggotaPilih:hover {
	background-color: #d1d1d1;
}</string>
        </property>
        <item>
         <property name="text">
          <string>Dosen/Karyawan</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Co Ass</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Mahasiswa</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QPushButton" name="kembaliButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#kembaliButton {
	color: #fcf9f4;
	border-radius: 5px;
	background-color: rgba(234, 118, 89, 220);
	font-weight: bold;
	height: 40px;
	font: 12pt &quot;Comic Sans MS&quot;;
}
QPushButton#kembaliButton:hover {
	background-color: rgba(234, 118, 89, 255);
	font: 14pt &quot;Comic Sans MS&quot;;
}</string>
        </property>
        <property name="text">
         <string>KEMBALI</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="simpanButton">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton#simpanButton {
	color: #fcf9f4;
	border-radius: 5px;
	background-color: rgba(234, 118, 89, 220);
	font-weight: bold;
	height: 40px;
	font: 12pt &quot;Comic Sans MS&quot;;
}
QPushButton#simpanButton:hover {
	background-color: rgba(234, 118, 89, 255);
	font: 14pt &quot;Comic Sans MS&quot;;
}</string>
        </property>
        <property name="text">
         <string>SIMPAN EDIT</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>noIDEdit</tabstop>
  <tabstop>namaEdit</tabstop>
  <tabstop>alamatEdit</tabstop>
  <tabstop>getFoto</tabstop>
  <tabstop>tipeAnggotaPilih</tabstop>
  <tabstop>simpanButton</tabstop>
  <tabstop>kembaliButton</tabstop>
 </tabstops>
 <resources>
  <include location="misc.qrc"/>
 </resources>
 <connections/>
</ui>
